src/HypothesisTests/AndersonRubinTest.jl

```diff
@@ -0,0 +1,14 @@
+"""
+    AR_gmm_obj(gi:Function)
+
+Returns a function `ar(θ)` where:
+``ar(θ) = n [1/n \\sum g_i(θ)]' \\widehat{Var}(g_i(θ))^{-1}[1/n \\sum g_i(θ)] ``
+"""
+function AR_gmm_obj(gi::Function)
```

Comment:
I think this is the same as `cue_objective`. It might be better to use `cue_objective` in its place.